src/routes/index_routes.js

```diff
@@ -14,7 +14,9 @@ routes.get('/',(req,res)=>{
 
 
         res.json(rows);
```

Comment:
```suggestion
        res.json(rows);
```
'ta mal e'ta vaina. que god es la totona

Reply:
si 'ta bien, cabeza 'e webo......
```suggestion
        res.json(rows); 8=====D
        // ta re lindo el codigo
```

Reply:
monda acrobatica